### ISSUE TITLE
No sounds when playing on a

### ISSUE THREAD

**Issue by MrRCE** [howardchung/jeopardy#66]

**Comment by howardchung** [comment]
No sounds meaning just no reading clues or also no daily double, timeout, etc sounds?

**Comment by MrRCE** [comment]
Yeah. No sounds at all on Ipad or IPhone. We also used different browsers like chrome, safari, Firefox etc. But it's the same. Do you know what's the problem?

**Comment by howardchung** [comment]
All iOS devices actually use Safari's engine behind the scenes--it may not have support for some advanced features like text to speech. The audio clips should be working though since Safari supports the Web Audio API: https://caniuse.com/audio-api

**Comment by howardchung** [comment]
Although apparently Safari should support speech synth as well: https://caniuse.com/?search=Web%20Speech%20API

**Comment by MrRCE** [comment]
![IMG_7754](https://github.com/howardchung/jeopardy/assets/142215756/fabc711e-bee5-4bdc-9b19-28d79c11d2dd)

**Comment by MrRCE** [comment]
It looks like this in an Iphone device when you open a game. No introduction sounds and no reading of the categories. The categories’ names also doesnt show unless you refresh it

**Comment by howardchung** [comment]
Yeah it does appear that iOS has weird restrictions bugs with audio, e.g. https://adactio.medium.com/web-audio-api-update-on-ios-1e553fff7847

But I should fix the category thing--the issue right now is that it's timed to show each category as it finishes reading it, so we need to handle the case where the reading doesn't work

**Comment by MrRCE** [comment]
Thanks for looking into this. You're app is absolutely perfect tho

**Comment by howardchung** [comment]
Fixed the category issue, but the no audio issue still remains
